viewer/core/src/utilities/Spinner.ts

```diff
@@ -6,10 +6,18 @@ import css from './spinnerStyles.css';
 import svg from '!!raw-loader!./spinnerCogniteLogo.svg';
 import * as THREE from 'three';
 
+import { assertNever } from '@reveal/utilities';
+
+export type Corner = 'topLeft' | 'topRight' | 'bottomLeft' | 'bottomRight';
```

Comment:
What is the benefit of this vs. an enum?

Reply:
It matches `Cognite3dViewerOptions` and how we've done it there - e.g. for MSAA and SSAO